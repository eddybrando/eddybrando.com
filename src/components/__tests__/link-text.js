import React from 'react';
import renderer from 'react-test-renderer';

import LinkText from '../link-text';

describe('LinkText', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <LinkText label="Contact me" to="/contact-me" style={{margin: 20}} />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
