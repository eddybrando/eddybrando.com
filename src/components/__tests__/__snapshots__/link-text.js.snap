// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkText renders correctly 1`] = `
<a
  className="link-block"
  href="/contact-me"
  style={
    Object {
      "margin": 20,
    }
  }
  title="Contact me"
>
  Contact me
   
  <svg
    aria-hidden={true}
    className="link-arrow"
    focusable="false"
    height={16}
    preserveAspectRatio="xMidYMid meet"
    style={
      Object {
        "willChange": "transform",
      }
    }
    viewBox="0 0 16 16"
    width={16}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9.3 3.7L13.1 7.5 1 7.5 1 8.5 13.1 8.5 9.3 12.3 10 13 15 8 10 3z"
    />
  </svg>
</a>
`;
